js/renderBot.js

```diff
@@ -166,12 +166,43 @@ function renderBot(directLineKey, botSecret) {
         }
     });
 
+    // calling session storage data to other pages on Bot 
+    $(function storeddata() {
+        if (sessionStorage["myKey"] != null) {
+            var contentsOfOldDiv = JSON.parse(sessionStorage["myKey"]);
+            $(".wc-message-group-content").html(contentsOfOldDiv);
+        }
+    });
+
+    $(function appendmessage() {
+        var a;
+        newMessages = document.getElementsByClassName("wc-message-groups")[0];
+        newMessages.addEventListener('DOMNodeInserted', function (evt) {
+            if (newMessages.getElementsByClassName("wc-message-group-content") !== undefined) {
+                var msgList = document.getElementsByClassName('wc-message-wrapper list').length;
```

Comment:
Use either Jquery or JavaScript consistently

Reply:
Used JavaScript and changed accordingly